File: sort0/sort.c

// Starter Code
#include <stdio.h>
//#include<stdio.h>
#include <stdlib.h>

// #include <stdio.h>
#include <string.h>
// Starter Code


int* sort_array(int arr[],int n)
{
    static int sorted_array[100000];
    //Write your code here without removing the exising code
	//the variable 'arr' contains array of integers of size n.
	//modified the array 'sorted_array' in sorted order of 0,1,2.
    return sorted_array;
}


//Backend Code
int main()
{
    int *arr;
    char str[100000];

    int n;
    scanf("%d", &n);
    int i = 0;

    arr = (int *)malloc(n * sizeof(int));
    int b[100];

    scanf("%s", str);
    char *token = strtok(str, "[");
    token = strtok(token, "]");
    token = strtok(token, ",");
    if (i < n)
    {
        while (token != NULL)
        {
            char *temp = token;
            int x = atoi(temp);
            *(arr + i) = x;
            token = strtok(NULL,",");
            i++;
        }
    }
   
    int *ans=sort_array(arr,n);
    for (int i = 0; i<n; i++)
    {
        
        printf("%d ",*(ans+i));
    }

    return 0;
}

File: sort0/sort.py
# Starter Code
class Solution:
    def sort_array(self, arr, n):
        sorted_array=[0]
        # Write your code here without removing the exising code
        # the variable 'arr' contains array of integers of size n.
        #modified the array 'sorted_array' in sorted order of 0,1,2.
        return sorted_array


if __name__ == '__main__':
    n = int(input())
    str = input()
    arr = [int(int(i)) for i in str.strip('[]').split(',')]
    x = Solution()
    ans=x.sort_array(arr, n)
    for i in range(0, len(ans)):
        print(ans[i], end=" ")
